<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExperimentView</class>
 <widget class="QWidget" name="ExperimentView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="maximumSize">
          <size>
           <width>390</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Отправка команд</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Диаметр (мм)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spinBoxDiameter">
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>71.400000000000006</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Имя</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leNameSpecimen">
              <property name="text">
               <string>test1</string>
              </property>
              <property name="placeholderText">
               <string>name</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QSpinBox" name="spinPressure">
              <property name="maximum">
               <number>2000</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>time</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>stabilization</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>manual</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spinCriter">
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="dateTimeEdit">
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="specialValueText">
               <string/>
              </property>
              <property name="dateTime">
               <datetime>
                <hour>0</hour>
                <minute>0</minute>
                <second>15</second>
                <year>2009</year>
                <month>1</month>
                <day>1</day>
               </datetime>
              </property>
              <property name="date">
               <date>
                <year>2009</year>
                <month>1</month>
                <day>1</day>
               </date>
              </property>
              <property name="time">
               <time>
                <hour>0</hour>
                <minute>0</minute>
                <second>15</second>
               </time>
              </property>
              <property name="currentSection">
               <enum>QDateTimeEdit::HourSection</enum>
              </property>
              <property name="displayFormat">
               <string notr="true">HH:mm:ss</string>
              </property>
              <property name="timeSpec">
               <enum>Qt::LocalTime</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnAddStep">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>42</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollLayout">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>312</width>
               <height>310</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="layoutSteps">
                <property name="spacing">
                 <number>0</number>
                </property>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>312</width>
               <height>310</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QTextEdit" name="textEditLogOut">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnClearLogOut">
                <property name="text">
                 <string>clear</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabGraphics">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="tabPosition">
          <enum>QTabWidget::East</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Page</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>GroupBox</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QPushButton" name="btnSetSettings">
                <property name="text">
                 <string>setSettings</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>setExperiment</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="btnStart">
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="btnInitStoreData">
                <property name="text">
                 <string>initStoreData</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="btnGetStoreData">
                <property name="text">
                 <string>getStoreData</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QPushButton" name="btnClearDataStore">
                <property name="text">
                 <string>clearData</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QPushButton" name="btnStopStoreData">
                <property name="text">
                 <string>stopStoreData</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="btnCreateGraph">
                <property name="text">
                 <string>graph</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="btnSaveImage">
                <property name="text">
                 <string>save image</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QPushButton" name="btnMoveUp">
                <property name="text">
                 <string>moveUp</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QPushButton" name="btnMoveDown">
                <property name="text">
                 <string>moveDown</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QPushButton" name="btnStopStepper">
                <property name="text">
                 <string>stop</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QPushButton" name="btnUnlockPid">
                <property name="text">
                 <string>unlockPID</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QPushButton" name="btnSetTarget">
                <property name="text">
                 <string>setTarget</string>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="QDoubleSpinBox" name="spinTargetMin">
                <property name="maximum">
                 <double>1000000.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>10.000000000000000</double>
                </property>
                <property name="value">
                 <double>300.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="5" column="3">
               <widget class="QDoubleSpinBox" name="spinTarget">
                <property name="minimum">
                 <double>5.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>2000.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>50.000000000000000</double>
                </property>
                <property name="value">
                 <double>500.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QPushButton" name="btnSetHz">
                <property name="text">
                 <string>setHz</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QDoubleSpinBox" name="spinHz">
                <property name="minimum">
                 <double>0.010000000000000</double>
                </property>
                <property name="maximum">
                 <double>5.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
                <property name="value">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QPushButton" name="btnHardReset">
                <property name="text">
                 <string>hardReset</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QPushButton" name="btnSetPidP">
                <property name="text">
                 <string>set down PID_P</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QDoubleSpinBox" name="spinSetPID_P">
                <property name="maximum">
                 <double>1000.000000000000000</double>
                </property>
                <property name="value">
                 <double>10.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="8" column="2">
               <widget class="QPushButton" name="btnSetPID_D">
                <property name="text">
                 <string>set down PID_D</string>
                </property>
               </widget>
              </item>
              <item row="8" column="3">
               <widget class="QDoubleSpinBox" name="spinSetPID_D">
                <property name="maximum">
                 <double>1000.000000000000000</double>
                </property>
                <property name="value">
                 <double>30.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QPushButton" name="btnSetUpPidP">
                <property name="text">
                 <string>set up PID_P</string>
                </property>
               </widget>
              </item>
              <item row="9" column="1">
               <widget class="QDoubleSpinBox" name="spinSetUpPID_P">
                <property name="maximum">
                 <double>1000.000000000000000</double>
                </property>
                <property name="value">
                 <double>10.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="9" column="2">
               <widget class="QPushButton" name="btnSetUpPID_D">
                <property name="text">
                 <string>set up PID_D</string>
                </property>
               </widget>
              </item>
              <item row="9" column="3">
               <widget class="QDoubleSpinBox" name="spinSetUpPID_D">
                <property name="maximum">
                 <double>1000.000000000000000</double>
                </property>
                <property name="value">
                 <double>30.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="10" column="0">
               <widget class="QPushButton" name="btnSetState">
                <property name="text">
                 <string>set state pid</string>
                </property>
               </widget>
              </item>
              <item row="10" column="1">
               <widget class="QComboBox" name="cmbState">
                <item>
                 <property name="text">
                  <string>P_correct</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>controller_default</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="11" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="4" column="0" colspan="4">
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>сенсоры</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QPushButton" name="btnGetSensorValue">
                   <property name="text">
                    <string>getSensorValue</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btnSensorSetZero">
                   <property name="text">
                    <string>Выставить в ноль</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cmbSensorName">
                   <item>
                    <property name="text">
                     <string>SensLoad0</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>SensLoad1</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>SensDef0</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>SensDef1</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>SensPrs2</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>SensPrs1</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>SensPrs0</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>SensBase</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="lblSensors">
             <property name="text">
              <string>Out sensors</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnClearTextEdit">
             <property name="text">
              <string>clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
